### PR TITLE
Delete Leaflet-offline

### PR DESCRIPTION
**Pull request by BETOXL**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## marked ([<kbd>unpkg</kbd>](https://unpkg.com/browse/marked@latest/))
was missing the following properties:
1. defaults
1. getDefaults
1. lexer
1. options
1. parse

<details>
<summary>as well as these 5 other properties...</summary>
<p>parseInline, parser, setOptions, use, walkTokens</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9f106fd0224dac249d1ed8cbe52e9457d1425516
</p>

**Comment by BETOXL** [comment]
Delete typed leaflet-offline

**Comment by BETOXL** [comment]
hope the @types/leaflet-offline can be removed and the new @types/ngx.leaflet.offline created

**Comment by amcasey** [comment]
@BETOXL can you please provide some more context?  GH seems to be having trouble with the diff because the merge-base is so old.  Does the underlying package, leaflet-offline, still exist?  The replacement (?) package appears to be yours - did you consider publishing the types there?  Thanks!

**Comment by BETOXL** [comment]
leaflet-offline does not exist, so remove the @types/leaflet-offline. On the other hand create a new package called ngx.leaflet.offline and I want to add its @types/ngx.leaflet.offline. Thanks a lot

File: types/ngx.leaflet.offline/tslint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-object-literal-type-assertion": false,
```
Comment (amcasey) [inline types/ngx.leaflet.offline/tslint.json:4]: If these are required, can you please suppress them on the lines where they occur.

**Review by amcasey — CHANGES_REQUESTED** [review]

**Review by BETOXL — COMMENTED** [review]
ok

File: types/ngx.leaflet.offline/tslint.json (hunk 1 above)
Comment (BETOXL) [reply types/ngx.leaflet.offline/tslint.json:4]: ok

Comment (BETOXL) [reply types/ngx.leaflet.offline/tslint.json:4]: if they are necessary.

**Review by BETOXL — COMMENTED** [review]
All ok

File: types/ngx.leaflet.offline/tslint.json (hunk 1 above)
Comment (amcasey) [reply types/ngx.leaflet.offline/tslint.json:4]: Can you please remove these entries?

**Pull request closed without merging**